ui-components/src/locales/bn.json

```diff
@@ -1,1263 +1,1264 @@
 {
```

Comment:
Do you know what caused these diffs in `bn.json` and `ar.json`? Is it maybe a prettier setting you have or a tab size setting?